pkg/result/processors/autogenerated_exclude.go

```diff
@@ -39,6 +39,10 @@ type fileSummary struct {
 	generated bool
 }
 
+// AutogeneratedExclude filters generated files.
+// - mode "lax": see [isGeneratedFileLax] documentation.
```

Comment:
`pkgsite` doesn't render documentation links for unexported functions, so having `[]` is redundant.

Also, please fix doc generation for the list items:

<img width="1016" alt="image" src="https://github.com/user-attachments/assets/b51d412b-9cf1-43cf-bf09-ba1a0c681f0c" />

http://localhost:8080/github.com/golangci/golangci-lint@v0.0.0/pkg/result/processors#AutogeneratedExclude